Microsoft.Identity.Web/Client/TokenAcquisition.cs

```diff
@@ -164,7 +164,20 @@ public async Task<string> GetAccessTokenOnBehalfOfUser(HttpContext context, IEnu
 
             // Use MSAL to get the right token to call the API
             var application = BuildConfidentialClientApplication(context, context.User);
-            return await GetAccessTokenOnBehalfOfUser(application, context.User, scopes, tenant);
+
+            // Case of a lazy OBO
+            Claim jwtClaim = context.User.FindFirst("jwt");
```

Comment:
not sure if this is great from a security point of view to add the JWT used to call the Web API as a claim?